contracts/core/chainlink-oracle/v1/ChainlinkOracle.sol

```diff
@@ -1,13 +1,13 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity 0.6.11;
 
-import "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+import { Ownable } from "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
```

Comment:
Could someone give me the reason for these explicit imports?

Reply:
You have a clear understanding of what is being imported and where it is being imported from. Without this, you have no idea where `SomeContract` is being imported from, or which version of `SomeContract` you have imported, if it's an inherited import, and you've imported 10 files, each of which imported 10 files, etc, etc. Consider in js:

```js
require('something`);
``

vs

```js
const { A, B } = require('something`);
``